### ISSUE TITLE
Cloning pages with seperate user name and password page.

### ISSUE THREAD

**Issue by iobj** [gophish/gophish#879]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 

Brief description of the issue: 4

What are you expecting to see happen? : When cloning the new Gmail page, for example, you input your Gmail account, then you have to hit "next" to get to the next page to enter your password.

What are you seeing happen? After cloning the page the only page rendered is the first-page requiring the username. GoPhish does not clone the password page after the victim hits next?

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```

Please provide any terminal output that may be relevant below:

```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - Clone the Gmail page, once this is done and the "victim" inputs their username into the first page and hit next it automatically takes them to the specified redirect page. It will not take them to the next page asking for their password.

Thanks!
 * Step 2 -

**Comment by jordan-wright** [comment]
I'm afraid there's no easy way for Gophish to be smart enough to know to clone multiple pages for a complete login flow. Some login flows require multiple pages, so you'll need to recreate these manually or change the flow to be a single page manually.

Either way, I'm afraid this is one of those things we can't do automatically :/
